python/templates/Collection.h.jinja2

```diff
@@ -48,6 +48,7 @@ A Collection is identified by an ID.
 */
 class {{ class.bare_type }}Collection : public podio::CollectionBase {
 public:
+  using object = {{ class.bare_type }};
```

Comment:
```suggestion
  using value_type = {{ class.bare_type }};
```
Then the renaming `object` -> `object_` should also become obsolete.

Do we need / want the `Mutable` version here as a member type.

Reply:
My assumption was that coll::value_type::mutable_type would be sufficient.